app/controllers/menus_controller.rb

```diff
@@ -2,6 +2,6 @@ class MenusController < ApplicationController
 
   def show
     @restaurant = Restaurant.find(params[:restaurant_id])
-    @cart_nr = ShoppingCart.count
+    @cart_nr = ShoppingCartItem.count
```

Comment:
Displays the number of items in cart on menus show page instead of number of carts as it did before. We are going to move this "counter" to the nav bar when that is completed.